config-service-change-event-api/src/main/proto/org/hypertrace/config/change/event/v1/config_change_event_key.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.config.change.event.v1;
+
+message ConfigChangeEventKey {
+  string tenant_id = 1;
+  string config_type = 2;
```

Comment:
Does it make these harder to use having to work with both keys and values on the consumer side? Seems like it could be a bit clunky, but not sure. I guess the alternative is to dupe all 3 into the value, which seems bad too.

Reply:
Should be fine in my view. No need to duplicate information in two places